src/Pages/Layout/Isna/Isna.js

```diff
@@ -0,0 +1,68 @@
+import Mousetrap from 'mousetrap';
+import React, { PureComponent } from 'react';
+import * as styles from './css/Isna.module.css';
+import Notifications from './data/Notifications';
+import IsnaPicture from './images/isna.png';
+import IsnaSound from './sounds/isna.mp3';
+import AsyncImage from '../../Utils/AsyncImage';
+
+class Isna extends PureComponent {
+  state = {
+    audio: new Audio(IsnaSound),
```

Comment:
This shoudn't be in state as rendering doesn't depend on it, consider moving it to an instance prop